app/components/windows/EditTransform.tsx

```diff
@@ -21,52 +24,71 @@ const dirMap = (dir: string) =>
 export default class EditTransform extends TsxComponent<{}> {
   @Inject() selectionService: SelectionService;
   @Inject() windowsService: WindowsService;
+  @Inject() private editorCommandsService: EditorCommandsService;
+
+  selection = this.selectionService.getActiveSelection();
 
   // We only care about the attributes of the rectangle not the functionality
-  originRect = { ...this.selectionService.getBoundingRect() };
-  rect = { ...this.selectionService.getBoundingRect() };
+  originRect = { ...this.selection.getBoundingRect() };
+  rect = { ...this.selection.getBoundingRect() };
 
   $refs: {
     validForm: ValidatedForm;
   };
 
   get transform() {
-    return this.selectionService.getTransform();
+    return this.selection.getItems()[0].transform;
   }
 
-  setTransform(key: string, subkey: string) {
+  setCrop(cropEdge: string) {
```

Comment:
It's a limited amount of Edges, right?  cropEdge: 'top' | 'right' | etc..

Reply:
Yes, would be more correct to have this be `keyof ICrop` I think.